snowflake_blueprints/upload_file.py

```diff
@@ -264,29 +222,9 @@ def put_csv(db_connection, file_path, table_name, results_dict):
             "uploaded_bytes": item[3],
             "input_file_type": item[4],
             "uploaded_file_type": item[5],
-            "status": item[6]}
-        results_dict['put'].append(put_results)
-
-    return results_dict
```

Comment:
Just so happens the next function that was removed had the same lines. These lines were not deleted but show up as changed after the remove of the execute_put_command